wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -2,10 +2,142 @@
 /*
 Plugin Name:  AB Content Alterer
 Description:  WordPress Content Alterer: Change Unbounce and Leadpages in content to <strong>Instapage</strong>
-Version:      0.1.0
+Version:      0.1.1
 Author:       andrzej.bialowas@instapage.com
 */
 defined('ABSPATH') or die('No script kiddies please!');
+
+class abContentAlererSettingsPage {
+    /**
+     * Holds the values to be used in the fields callbacks
+     */
+  private $options;
+
+  /**
+   * Start up
+   */
+  public function __construct() {
+    add_action('admin_menu', array( $this, 'add_plugin_page'));
+    add_action('admin_init', array( $this, 'page_init'));
+  }
+
+  /**
+   * Add options page
+   */
+  public function add_plugin_page() {
+    // This page will be under "Settings"
+    add_options_page(
+      'Settings Admin', 
+      'AB Content Alterer Settings', 
+      'manage_options', 
+      'ab-settings-admin', 
+      array($this, 'create_admin_page')
+    );
+  }
+
+  /**
+   * Options page callback
+   */
+  public function create_admin_page() {
+    // Set class property
+    $this->options = get_option('ab_alterer_option_group');
+    ?>
+    <div class="wrap">
+      <h1>AB Content Alterer</h1>

```

Comment:
use __() function to output text. It will be easier to translate the plugin with that.